doc/source/management/data_at_rest_encryption.rst

```diff
@@ -80,6 +80,55 @@ Trying to add unencrypted table to this table space will result in an error:
   |Percona XtraBackup| currently doesn't support backup of encrypted general
   tablespaces.
 
+Binary log encryption
+=====================
+
+A new option, implemented since |Percona Server| :rn:`5.7.20-19`, is
+encryption of binary and relay logs, triggered by the
+:variable:`encrypt_binlog` variable.
+
+Besides turning :variable:`encrypt_binlog` ``ON``, this feature requires both
+`master_verify_checksum
+<https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_master_verify_checksum>`_
+and `binlog_checksum
+<https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_checksum>`_
+variables to be turned ``ON``.
+
+Distinct keys are used for binary and relay logs, and key identifiers include
+server UUIDs, which makes multi-source replication possible.
+The encryption key is always automatically rotated at the start of each
+binary log file. Therefore configuring the automatic purging of old binary
+logs ensures that old key versions are not in use.
+``PURGE BINARY LOGS`` and ``RESET MASTER`` commands delete obsolete binary log
+key versions from the keyring, and obsolete relay log key versions are deleted
+together with their corresponding relay logs. This scheme is independent from
+the master instance key and its rotation.
+
```

Comment:
Key rotation is not yet implemented. In current state the same key is used always. In the next iteration when key versioning gets implemented it will be possible to rotate binlog key with use of SQL function. In further iterationx we will add PURGE BINARY LOGS and RESET MASTER support together with rotating key on start of each binary log file (this is however problematic as might cause binlog to grow rapidly and cause some lags in case keyring_vault - each rotation will require connecting to Vault server).